ocean_lib/models/v4/test/test_swap_fees.py

```diff
@@ -0,0 +1,3906 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import pytest
+from web3 import exceptions
+import math
+from ocean_lib.models.v4.dispenser import DispenserV4
```

Comment:
please separate the external imports from our internal ones, through an empty line.
Also, please order each category alphabeticallly.